validators/kernel_validator.go

```diff
@@ -90,7 +90,7 @@ func (k *KernelValidator) validateKernelVersion(kSpec KernelSpec) error {
 		}
 	}
 	k.Reporter.Report("KERNEL_VERSION", k.kernelRelease, bad)
-	return fmt.Errorf("unsupported kernel release: %s", k.kernelRelease)
+	return fmt.Errorf("kernel release %s is unsupported, and %s", k.kernelRelease, kSpec.VersionsNote)
```

Comment:
The final error message is like`kernel release %s is unsupported, and kernel version should be in maintenance, at least 4.4 or 4.19+, and to use cgroup v2, 5.8+ is recommended.`

Reply:
i think we can make it
```suggestion
	return fmt.Errorf("kernel release %s is unsupported. %s", k.kernelRelease, kSpec.VersionsNote)
```

so just put `.` and after that is the note.